src/quo2/components/list_item.cljs

```diff
@@ -0,0 +1,250 @@
+    :danger   {:icon-color         (theme-colors colors/danger-50 colors/danger-60)
+               :active-background  (theme-colors colors/neutral-10 colors/neutral-90)
+               :passive-background (theme-colors colors/white colors/neutral-90)
+               :text-color         (theme-colors colors/danger-50 colors/danger-60)}))
+
+(defn size->icon-size [size]
+  (case size
+    :small 20
+    32))
+
+(defn size->container-size [size]
+  (case size
+    :small 48
+    60))
+
+(defn size->single-title-size [size]
+  (case size
+    :small :paragraph-1
+    :large :heading-2))
+
+(defn container [{:keys [size container-style]} & children]
+  (into [rn/view {:style (merge (:base spacing/padding-horizontal)
+                                {:min-height       (size->container-size size)
+                                 :padding-vertical 8
+                                 :flex-direction   :row
+                                 :align-items      :center
+                                 :justify-content  :space-between} container-style)}]
+        children))
+
+(defn icon-column
+  [{:keys [icon icon-color size icon-container-style]}]
+  (when icon
+    (let [icon-size (size->icon-size size)]
+      [rn/view {:style (or icon-container-style (:tiny spacing/padding-horizontal))}
+       (cond
+         (vector? icon)
+         icon
+
+         (keyword? icon)
+         [rn/view {:style {:width            icon-size
+                           :height           icon-size
+                           :align-items      :center
+                           :justify-content  :center
+                           :border-radius    (/ icon-size 2)}}
+          [icons/icon icon {:color icon-color}]])])))
+
+(defn title-column
+  [{:keys [title text-color subtitle subtitle-max-lines subtitle-secondary
+           title-accessibility-label size text-size title-text-weight
+           right-side-present?]}]
+  [rn/view {:style (merge (:tiny spacing/padding-horizontal)
+                          ;; make left-side title grow if nothing is present on right-side
+                          (when-not right-side-present?
+                            {:flex            1
+                             :justify-content :center}))}
+   (cond
+
+     (and title subtitle)
+     [:<>
+      ;; FIXME(Ferossgp): ReactNative 63 will support view inside text on andrid, remove thess if when migrating
+      (if (string? title)
+        [text/text {:weight              (or title-text-weight :medium)
+                    :style               {:color text-color}
+                    :accessibility-label title-accessibility-label
+                    :ellipsize-mode      :tail
+                    :number-of-lines     1
+                    :size                text-size}
+         title]
+        title)
+      (if (string? subtitle-secondary)
+        [rn/view {:flex-direction :row}
+         [text/text {:style {:max-width "56.5%"}
```

Comment:
oh no, magic numbers %(

Reply:
this was specific implementation of quo1 so i am removing it, probably it is not needed in quo2